### PR TITLE
Add stories for `OverlayMessage` + TS

### PR DESCRIPTION
**Pull request by haotongwang**
## Description

Title

## Screenshots

image

## Steps to Test

1. Run `yarn run storybook` in `dashboard` `client`

### CODE REVIEW

**Review by hallgchris — DISMISSED** [review]
Nice stuff, LGTM!

File: client/src/components/v2/OverlayMessage.tsx [hunk 1]
```diff
@@ -39,7 +41,7 @@ export default function OverlayMessage() {
             value={message}
           />
           <InputGroup.Append>
-            <Button variant="outline-secondary" onClick={sendMessage}>
+            <Button variant="outline-secondary" onClick={(_) => sendMessage(message)}>
```
Comment (hallgchris) [inline client/src/components/v2/OverlayMessage.tsx]: Not sure how I missed this before, but I'm getting a compile error on the unused variable `_` (which can just be removed entirely)

**Review by hallgchris — CHANGES_REQUESTED** [review]

File: client/src/components/v2/OverlayMessage.tsx (hunk 1 above)
Comment (Blake-Haydon) [reply client/src/components/v2/OverlayMessage.tsx]: Just removed the underscore really quickly. Should build now!

File: client/src/components/v2/OverlayMessage.tsx [hunk 2]
```diff
@@ -39,7 +41,7 @@ export default function OverlayMessage() {
             value={message}
           />
           <InputGroup.Append>
-            <Button variant="outline-secondary" onClick={sendMessage}>
+            <Button variant="outline-secondary" onClick={() => sendMessage(message)}>
```
Comment (Blake-Haydon) [inline client/src/components/v2/OverlayMessage.tsx]: This button does not clear the message text whereas pressing enter clears the text. Is there a reason for this difference? If not we should do something similar to this.
``` 
  const handleKeyPressed = useCallback(
    (event) => {
      if (event.key === 'Enter') {
        sendMessage(message);
        setMessage('');
      }
    }
```

**Review by Blake-Haydon — CHANGES_REQUESTED** [review]
Everything looks good apart from the inconsistency when pressing `enter` and the button clearing the text.

I also got a weird error when I saved while running the notebook. Any ideas?
image

File: client/src/components/v2/OverlayMessage.tsx (hunk 1 above)
Comment (hallgchris) [reply client/src/components/v2/OverlayMessage.tsx]: Generally don't commit to other people's branches unless you've communicated about it with the owner - things can get pretty messed up with multiple people working on a branch! It's one of the biggest reasons we have them at all

File: client/src/components/v2/OverlayMessage.tsx (hunk 2 above)
Comment (haotongwang) [reply client/src/components/v2/OverlayMessage.tsx]: Done
35a4e17

**Comment by haotongwang** [comment]
@Blake-Haydon It seems to build without errors on my side though I did have some issues with getting webpack/ts to use the `.eslintrc` config in the project. It looks like maybe it's using a global eslint config?

**Review by hallgchris — DISMISSED** [review]
I've had that same error appear sometimes and it's completely perplexed me - I assumed it was only me who got it. I've found that rerunning storybook made it go away for a short while though. Should try to work out what causes it though.

The error isn't caused by this PR though - Looks good to me!

**Review by Blake-Haydon — DISMISSED** [review]
Error was just on my side. LGTM.

**Review by hallgchris — APPROVED** [review]
@Blake-Haydon did you figure out how to fix it?

**Comment by Blake-Haydon** [comment]
> @Blake-Haydon did you figure out how to fix it?

I moved everything it said into a dev dependency and then it stopped fussing. Although working on my own branch now I'm getting a different error with storybook... So I'm still not sure the root cause. @hallgchris

**Comment by hallgchris** [comment]
That's weird. I'll try again, but I recall moving it to a dev dependency and still getting the error when I tried it a while ago. I'll have to try again.

**Pull request merged**